extensions/kusto/src/credentialstore/credentialstore.ts

```diff
@@ -0,0 +1,103 @@
+/*---------------------------------------------------------------------------------------------
+ *  Copyright (c) Microsoft Corporation. All rights reserved.
+ *  Licensed under the Source EULA. See License.txt in the project root for license information.
+ *--------------------------------------------------------------------------------------------*/
+
+import { SqlOpsDataClient, ClientOptions, SqlOpsFeature } from 'dataprotocol-client';
+import { IConfig, ServerProvider } from 'service-downloader';
+import { ServerOptions, RPCMessageType, ClientCapabilities, ServerCapabilities, TransportKind } from 'vscode-languageclient';
+import { Disposable } from 'vscode';
+import * as UUID from 'vscode-languageclient/lib/utils/uuid';
+import * as azdata from 'azdata';
+
+import * as Contracts from './contracts';
+import * as Constants from './constants';
+import * as Utils from '../utils';
+
+class CredentialsFeature extends SqlOpsFeature<any> {
```

Comment:
the credential store should already be defined elsewhere.  kusto should just be able to use it.

Reply:
The only other place this is defined is in the mssql extension. Do you think we should extract this out to a common place?